frontend/src/DmarcByDomainPage.js

```diff
@@ -115,6 +117,24 @@ export default function DmarcByDomainPage() {
     {
       Header: i18n._(t`Domain`),
       accessor: 'domain',
+      Cell: ({ value }) => {
+        return (
+          <Stack direction="row" fontSize="sm" justifyContent="space-between">
+            <Box alignSelf="center">
+              {value}
+            </Box>
+            <TrackerButton
+              variant="primary"
+              as={RouteLink}
+              to={`/domains/${value}/dmarc-report/LAST30DAYS/${new Date().getFullYear()}`}
+            >
+              <Text whiteSpace="noWrap">
+                <Trans>DMARC Report</Trans>
+              </Text>
+            </TrackerButton>
+          </Stack>
+        )
+      },
```

Comment:
I think the following code is a good compromise of being recognizable as a link but still remaining subtle. It should also fix the eslint issue.

```
      import { Link as RouteLink } from 'react-router-dom'
      
      // eslint-disable-next-line react/prop-types
      Cell: function CellValueWithLink({ value }) {
        return (
          <Link
            as={RouteLink}
            to={`domains/${value}/dmarc-report/LAST30DAYS/${new Date().getFullYear()}`}
            isExternal={false}
          >
            {`${value} `} <Icon name="link" />
          </Link>
        )
      },
```

![image](https://user-images.githubusercontent.com/47400288/120811516-6d67a100-c522-11eb-9760-5e8a76c9eebe.png)